### PR TITLE
[DNM] Sec Instructor

### PR DESCRIPTION
**Pull request by AffectedArc07**
# NOTE
**If you want to comment on the sec instructor __ITSELF__, __PLEASE__ go to the forum thread linked [HERE](https://nanotrasen.se/forum/topic/15704-sec-instructor-jobs-a-remedy-to-newsec/). If you have an actual __CODE COMMENT__, then comment here.**

# PR
**What does this PR do:**
Adds the security instructor, with some clothing for it. The instructor's job is to teach security how to do their job properly, so its not wasting the warden or HoS's time. The role is behind a 45 karma lock, similar to the magistrate, so that only experienced people can play it. Currently they spawn in the room where the rest of the officers spawn, as I feel an office for them is unneeded.

**Images of sprite/map changes:**
IconDiffBot will handle these, but will be posted if requested

**Changelog:**
:cl: AffectedArc07, concept by Furasian
add: Security instructor
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
And before anyone flames me for this, I am completley neutral on this and hold no opinion, and dont really care. I am just making this because I was asked. Opinion is about as valid as a grain of salt.

**Comment by Fox-McCloud** [comment]
Marking as controversial, as requested.

**Comment by TDSSS** [comment]
You should probably describe implementation details in the actual PR. Stuff like exact playtime requirement. I assume they don't even get their own locker with gear or anything, they just spawn with their special stuff?

**Comment by Ty-Omaha** [comment]
**"Drill Instructor"** sounds completely wrong, they aren't a drill instructor.

Field Training Officer or something else is more appropriate

**Comment by Couls** [comment]
Are you planning to map in an office for the instructor?

**Comment by AffectedArc07** [comment]
@Ty-Omaha @Couls those are questions for the thread, and my answers arent really useful since I am just doing the code for this, I have no say in the actual idea, but I can answer what I know.

@Couls currently they dont have an office because it doesnt seem required

**Comment by AffectedArc07** [comment]
@TDSSS updated

**Comment by Christasmurf** [comment]
Might be an idea to include screenshots of the uniforms in the PR, so people can see what those look like.

**Comment by AffectedArc07** [comment]
https://github.com/ParadiseSS13/Paradise/pull/11387/checks?check_run_id=112281292

@Christasmurf collapse the things here

**Comment by Christasmurf** [comment]
@AffectedArc07 ah I'm not very good at github cheers

**Comment by Spartan6** [comment]
I mean, if we really need another role for security, a dispatcher would've been a better idea.

But... ehhh? Suuure?

**Comment by AffectedArc07** [comment]
@Spartan6 can you comment that on the thread instead of here please?

**Comment by Ty-Omaha** [comment]
-1 until name addressed

**Comment by AffectedArc07** [comment]
@Ty-Omaha Same for you, thats about the role itself, not code related

**Comment by Ty-Omaha** [comment]
@AffectedArc07 I did, I'm just making it clear on here why I am giving a -1 when I support it otherwise.

File: code/game/jobs/job/security.dm [hunk 1]
```diff
@@ -259,4 +259,39 @@
...
+	title = "Drill Instructor"
+	flag = INSTRUCTOR
+	department_flag = KARMA
+	total_positions = 1
+	spawn_positions = 1
+	is_security = 1
+	supervisors = "the head of security"
+	department_head = list("Head of Security")
+	selection_color = "#ffeeee"
+	access = list(access_security, access_sec_doors, access_brig, access_court, access_maint_tunnels, access_morgue, access_weapons)
```
Comment (Ralta) [inline code/game/jobs/job/security.dm]: Why do they have morgue access? Pretty sure Sec Officers don't, so a Sec Instructor shouldn't need it either.

Comment (AffectedArc07) [reply code/game/jobs/job/security.dm]: /shrug/
Im just doing as I was told

File: code/modules/clothing/under/jobs/security.dm [hunk 2]
```diff
@@ -199,3 +199,20 @@
 	icon_state = "pod_pilot"
 	item_state = "pod_pilot"
 	item_color = "pod_pilot"
+
+//Drill Instructor
+/obj/item/clothing/under/rank/security/instructor
+	name = "drill instructor's jumpsuit"
+	desc = "A buttoned red shirt tucked into black slacks, it bears the Security Patch on it's sleeves"
```
Comment (Ralta) [inline code/modules/clothing/under/jobs/security.dm]: `its` not `it's`.

File: code/modules/clothing/under/jobs/security.dm [hunk 3]
```diff
@@ -199,3 +199,20 @@
...
+	name = "drill instructor's jumpsuit"
+	desc = "A buttoned red shirt tucked into black slacks, it bears the Security Patch on it's sleeves"
+	icon_state = "drillshirt"
+	item_state = "drillshirt"
+	item_color = "drillshirt"
+	flags_size = ONESIZEFITSALL
+
+/obj/item/clothing/under/rank/security/instructor/skirt
+	name = "drill instructor's jumpskirt"
+	desc = "A buttoned red shirt neatly lined up with a lower skirt, it bears the Security Patch on it's sleeves"
```
Comment (Ralta) [inline code/modules/clothing/under/jobs/security.dm]: `its` not `it's`.

File: code/modules/clothing/under/jobs/security.dm (hunk 2 above)
Comment (AffectedArc07) [reply code/modules/clothing/under/jobs/security.dm]: This was copypasted from the person that asked for it

File: code/modules/clothing/under/jobs/security.dm (hunk 3 above)
Comment (AffectedArc07) [reply code/modules/clothing/under/jobs/security.dm]: This was copypasted from the person that asked for it

File: code/game/jobs/job/security.dm (hunk 1 above)
Comment (Ralta) [reply code/game/jobs/job/security.dm]: Irrespective of my opinions on this PR, I can't see morgue access as being necessary, can you?

File: code/modules/clothing/under/jobs/security.dm (hunk 3 above)
Comment (Ralta) [reply code/modules/clothing/under/jobs/security.dm]: You don't need to defend it bruh, I'm just pointing out the issue.

**Comment by variableundefined** [comment]
I don't see the necessity for this role at all. 

In addition, as a PR author, you should be ready to make changes to your own PR as you're the one who own the code, not the person who actually made the idea. By redirecting all conflict resolution to a third party who don't post on github you make it harder for everyone to get a mutually acceptable PR out.

**Comment by AffectedArc07** [comment]
This is right at the bottom of priorities, to the point where I might hand this to someone else.

Current things more important than this PR:
- Finals in 2 weeks
- Subsystem reworks
- Parallax
- Things on Kyet's list

Not to mention this has a low merge chance, its fair to say this isnt at the top of my list

**Comment by variableundefined** [comment]
Closing the PR as the role is not necessary or wanted.

**Pull request closed without merging**